packages/itwin/property-grid/src/test/hooks/UseDataProvider.test.ts

```diff
@@ -0,0 +1,39 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+
+import { expect } from "chai";
+import sinon from "sinon";
+import { PresentationPropertyDataProvider } from "@itwin/presentation-components";
+import { renderHook } from "@testing-library/react-hooks";
+import { useDataProvider } from "../../property-grid-react";
+import { stubFavoriteProperties } from "../TestUtils";
+
+import type { IModelConnection } from "@itwin/core-frontend";
+
+describe("useDataProvider", () => {
+  const imodel = {} as IModelConnection;
+
+  class CustomProvider extends PresentationPropertyDataProvider {
+  }
+
+  before(() => {
+    stubFavoriteProperties();
+  });
+
+  after(() => {
+    sinon.restore();
+  });
+
+  it("creates default provider", async () => {
+    const { result } = renderHook(useDataProvider, { initialProps: { imodel } });
+    expect(result.current).to.be.instanceOf(PresentationPropertyDataProvider);
```

Comment:
I suppose this would succeed for `CustomProvider` as well. Maybe change to the following?
```suggestion
    expect(result.current).to.be.instanceOf(PresentationPropertyDataProvider).and.not.be.instanceOf(CustomProvider);
```